### PR TITLE
SLNS-74: Last flown core stats

### PR DESCRIPTION
**Pull request by JRascagneres**

### CODE REVIEW

File: api/models.py [hunk 1]
```diff
@@ -620,6 +620,65 @@ class FirstStage(models.Model):
...
+        last_launch = Launch.objects.filter(rocket__firststage__launcher__id=self.launcher.id).filter(net__lt=launch_net).order_by('-net').first()
+
+        if last_launch:
+            res = last_launch.id
+
+        cache.set(cache_key, res, CACHE_TIMEOUT_ONE_DAY)
+        return res
+
+    @property
+    def previous_flight_name(self):
```
Comment (ItsCalebJones) [inline api/models.py]: After reflection - I think the @property should be def previous_flight(self) and just return the launch object. That way the API or webside can use the full object as needed.

Comment (JRascagneres) [reply api/models.py]: Okay. You reckon replace name + ID with the full object then?

Comment (ItsCalebJones) [reply api/models.py]: Correct - full object - then have as serializer that's just name and ID for the API.

Comment (JRascagneres) [reply api/models.py]: Okay. Not back until later tonight but will do it.

**Comment by JRascagneres** [comment]
Ready for new review

**Pull request merged**